lib/workload/orcabus-stateless-stack.ts

```diff
@@ -41,10 +63,50 @@ export class OrcaBusStatelessStack extends cdk.Stack {
 
     // hook microservice construct components here
     this.createSequenceRunManager();
+
+    if (props.filemanagerDependencies) {
+      this.createFilemanager(props.filemanagerDependencies, this.vpc);
+    }
   }
 
   private createSequenceRunManager() {
     // TODO new SequenceRunManagerConstruct() from lib/workload/stateless/sequence_run_manager/deploy/component.ts
     //   However, the implementation is still incomplete...
   }
+
+  private createFilemanager(dependencies: FilemanagerDependencies, vpc: IVpc) {
+    // Opting to reconstruct the dependencies here, and pass them into the service as constructs.
```

Comment:
I have a feeling this pattern will be very common, so it might be good to have another class somewhere that does this exclusively, e.g. `DependencyReconstructor.ts`.